src/test/resources/htsjdk/samtools/cram/ambiguityCodes.fasta

```diff
@@ -0,0 +1,2 @@
+>Sheila
```

Comment:
can you test this over the entirety of 'ACGTacgMmRrWwSsYyKkVvHhDdBbNn.' in the reference **crossed with** ACGTN in the read (so 32x5 length in total).